modules/common/src/logger/logger.utils.ts

```diff
@@ -0,0 +1,8 @@
+import * as httpContext from 'express-http-context';
+import {REQUEST_UNIQUE_ID_KEY} from '../constants';
+
+export function getCurrentRequestUniqueId (): string {
+    const uniqueId: string = httpContext.get(REQUEST_UNIQUE_ID_KEY);
+
+    return uniqueId || '';
```

Comment:
why do you default here to '' should you default also to uuidv4?

Reply:
the generation here could lead into multiple uuids in one request